packages/form-generator/src/local_codegen_graphql_form_generator.test.ts

```diff
@@ -251,4 +254,36 @@ void describe('LocalCodegenGraphqlFormGenerator', () => {
     const forms = await l.generateForms();
     assert('writeToDirectory' in forms);
   });
+  const graphqlDirectories = [
```

Comment:
@alharris-at @sobolk I added these tests which exercise the path generation logic. This is integration-level; it is calling the real codegen package and making assertions against the output.

Reply:
<img width="1023" alt="image" src="https://github.com/aws-amplify/amplify-backend/assets/18506410/0c95aa6e-aef4-410b-a6c2-5ca15dba293f">
These tests are passing on windows